diag_manager/fms_doubly_linked_list.F90

```diff
@@ -0,0 +1,275 @@
+!> \author Miguel Zuniga
```

Comment:
Needs the license and the doxygen file headers

Reply:
Done.